### PR TITLE
Bl 8518 autocomplete js assets

### PR DESCRIPTION
**Pull request by smcveigh941**
# New PR ~ front end

***

**Before submitting a PR, ensure** - 

- [ ] All Styleguide components remain functional.
- [ ] CSS updates did not unwillingly affect styleguide components. 
- [ ] Unnecessary console logs are removed.
- [ ] Unnecessary white space is removed.
- [ ] Code has been indented correctly.
- [ ] You haven't added inline styles.
- [ ] You haven't added inline JavaScript.
- [ ] Your changes (if any significant) have been cross browser tested.
- [ ] Another FED has reviewed.

* Mark done with an 'X' *

***

## Notes

_If there is something worth mentioning..._

### CODE REVIEW

File: src/assets/scss/gds/template/partials/_fonts.scss [hunk 1]
```diff
@@ -33,10 +33,10 @@
...
-  src: url('#{$fonts-dir}dotsfont.eot?#iefix') format('embedded-opentype'),
-       url('#{$fonts-dir}dotsfont.woff') format('woff'),
-       url('#{$fonts-dir}dotsfont.ttf') format('truetype'),
-       url('#{$fonts-dir}dotsfont.svg#dotsfontregular') format('svg');
+  font-family: 'text-security-disc';
+  src: url('#{$fonts-dir}text-security-disc.eot');
+  src: url('#{$fonts-dir}text-security-disc.eot?#iefix') format('embedded-opentype'),
+  url('#{$fonts-dir}text-security-disc.woff') format('woff'),
+  url('#{$fonts-dir}text-security-disc.ttf') format('truetype'),
+  url('#{$fonts-dir}text-security-disc.svg#text-security') format('svg');
```
Comment (MartinDM) [inline src/assets/scss/gds/template/partials/_fonts.scss]: The SVG file prevents build, and is not really a required format.
Webpack will error when running `npm build production` when parsing this SVG.
Please remove the reference to it and the .svg font.

File: src/assets/scss/mts/modules/form/partials/_overrides.scss [hunk 2]
```diff
@@ -8,15 +8,22 @@
 }
 
 .password-field {
-  -webkit-text-security: disc; 
+  caret-color: black;
+  -webkit-text-security: disc;
```
Comment (MartinDM) [inline src/assets/scss/mts/modules/form/partials/_overrides.scss]: The new font means we could remove this vendor prefix, as it looks very slightly, but not significantly different to the font implementation.
If we keep it in,  are we happy with the difference?
`caret-color` also is not supported in IE; does the colour fall back appropriately in IE?

File: src/assets/scss/gds/template/partials/_fonts.scss (hunk 1 above)
Comment (MartinDM) [inline src/assets/scss/gds/template/partials/_fonts.scss]: The SVG file prevents build, and is not really a required format.
Webpack will error when running npm build production when parsing this SVG.
Please remove the reference to it and the .svg font.

File: src/assets/scss/mts/modules/form/partials/_overrides.scss (hunk 2 above)
Comment (smcveigh941) [reply src/assets/scss/mts/modules/form/partials/_overrides.scss]: Yes, the caret looks fine in IE. I also agree with removing -webkit-text-security

**Comment by smcveigh941** [comment]
Should be all good to go now if you are @MartinDM :)

**Comment by MartinDM** [comment]
Great stuff! Thanks Stephen

**Comment by smcveigh941** [comment]
Fixed Merge Conflicts too

**Pull request merged**